src/i18n/i18n.service.ts

```diff
@@ -34,8 +34,30 @@ export const replace = (subject, variables) => subject.pipe(
 	})
 );
 
+// custom deep object merge
+const merge = (target, ...objects) => {
```

Comment:
Maybe add this to `src/utils/utils.ts` or `src/common/utils.ts` instead?

Reply:
👍